ci-operator/step-registry/fips-check/node-scan/fips-check-node-scan-commands.sh

```diff
@@ -96,10 +96,9 @@ EOF
 else
     cat >"${ARTIFACT_DIR}/${filename}.xml" <<EOF
     <testsuite name="${testsuite}" failures="1" errors="0" skipped="0" tests="1" time="$SECONDS">
-        <testcase name="${subteam}:Node scan of fips check should succeedded or skipped"/>
```

Comment:
Sorry did not catch the format issue in last review. And did not catch the issue of missing "|| true". Could such issues be earlier found during pj-rehearse tests :) ?

Reply:
Yes. When it fails, the whole job will be blocked. But it not always reproducible.